Running test on mutant:
============================= test session starts ==============================
platform linux -- Python 3.8.19, pytest-8.3.3, pluggy-1.5.0
rootdir: /mnt/temp/inspect_mutant
collected 1 item

../test.py F

=================================== FAILURES ===================================
_____________________________________ test _____________________________________

    def test():
        # for similar values a-b == a%b, so we patch the timing function to start at 0 again.
        # this isn't strictly necessary to observe mutant behavior.
        import time
        start_time = time.perf_counter()
        original_function = time.perf_counter
        time.perf_counter = lambda: original_function() - start_time
    
        log = []
    
        @Timer(text="{seconds}", logger=log.append)
        def timed_fun():
            time.sleep(.1)
            pass
    
        timed_fun()
        seconds = float(log[0])
    
>       assert abs(seconds - .1) < 0.01
E       assert 0.09999846029095352 < 0.01
E        +  where 0.09999846029095352 = abs((1.5397090464830399e-06 - 0.1))

../test.py:22: AssertionError
=========================== short test summary info ============================
FAILED ../test.py::test - assert 0.09999846029095352 < 0.01
============================== 1 failed in 0.13s ===============================
